templates/engage/unlisted/at&t-onboarding.html

```diff
@@ -23,232 +23,193 @@
+      <h2>Welcome AT&amp;T users!</h2>
+      <p>Canonical, the publisher of Ubuntu, has created an Ubuntu Pro Private Offer for AT&amp;T on Azure to provide custom images carefully optimised to work at scale for your service delivery architectures and IT workloads that will be moving to the public cloud.</p>
     </div>
   </div>
 </section>
 
 <section class="p-strip">
   <div class="row">
     <div class="col-7">
-      <h2 class="p-heading--3">Why Ubuntu Pro?</h2>
-      <p>This private offer allows AT&T's Azure and AWS subscription owners to receive security updates, security certifications, and support Ubuntu with custom pricing and SLAed support from Canonical.</p>
-      <p>Ubuntu Pro for Azure is a premium image designed by Canonical for AT&T that provides additional coverage for AT&T production workloads running in Azure. Ubuntu Pro includes security and compliance services, enabled to you by default and suitable for linux enterprise operations, with metered, pay-as-you-go billing on your existing Azure invoice.</p>
-      <p>Ubuntu Pro is a production-focused custom image which includes a 10 year commitment from Canonical to provide the broadest security coverage in the industry plus certified components for compliance and hardening, and Kernel Livepatch.</p>
+      <h2>Have specific project requirements?</h2>
+      <p>To speak about a specific project requirement, or have any questions, please contact us:</p>
+      <ul>
+        <li><strong>Rick Hawk</strong> (AT&amp;T Account Director) &ndash; <a href="mailto:rick.hawk@canonical.com">rick.hawk@canonical.com</a></li>
+        <li><strong>Canonical Support</strong> &ndash; <a href="mailto:support@canonical.com">support@canonical.com</a></li>
+        <li><strong>Canonical's Customer Success Team</strong> &ndash; <a href="mailto:customersuccess@canonical.com">customersuccess@canonical.com</a></li>
+      </ul>
+    </div>
+    <div class="col-5 u-hide--medium u-hide--small u-align--center u-vertically-center u-hide--medium u-hide--small">
+      {{
+      image(
+      url="https://assets.ubuntu.com/v1/24a03775-Contact+us.svg",
+      alt="",
+      height="171",
+      width="240",
+      hi_def=True,
+      loading="lazy",
+      ) | safe
+      }}
     </div>
   </div>
 </section>
 
 <section class="p-strip--light">
-  <div class="row">
-    <div class="col-7">
-      <h2 class="p-heading--3">AT&T Pricing</h2>
-      <p>The Ubuntu Pro private offer includes custom pricing. AT&T can add Ubuntu Pro using the custom SKU in Azure for only $0.004 (less than half a penny) per core per hour.</p>
+  <div class="u-fixed-width">
+    <h2>Why an Ubuntu Pro Private Offer for Azure?</h2>
+  </div>
+  <div class="row" style="padding-top: 60px; padding-bottom: 60px;">
```

Comment:
Maybe it's for design review but are paddings necessary here?
The space looks a bit wide between "Why an Ubuntu Pro Private Offer for Azure?" and "Security and SLA Support" 
<img width="790" alt="image" src="https://user-images.githubusercontent.com/57550290/192309621-f35cc6b3-04fa-4eca-9a9b-8783a494adfa.png">

Reply:
even if it is a design choice, inline px values are hardly the best way to do this. Can you please use a strip instead.